migration-engine/migration-engine-tests/tests/native_types/mysql.rs

```diff
@@ -847,7 +874,7 @@ fn risky_casts_with_existing_data_should_warn(api: TestApi) {
     }
 }
 
-#[test_connector(tags(Mysql))]
+#[test_connector(tags(Mysql), exclude(TiDB))]
```

Comment:
This excludes TiDB now. Are the other changes in this file, line 890 and ~756 not necessary any more then?
What is the limitation that makes it necessary to exclude TiDB here?

Reply:
This is caused by another issue: https://github.com/pingcap/tidb/issues/38568

In this test case, Prisma will execute the following DDL, which is to drop and add the column with the same name in one statement.

```sql
ALTER TABLE `Test` DROP COLUMN `col0`,
    ADD COLUMN `col0` DECIMAL(15, 6) NULL,
    DROP COLUMN `col1`,
    ADD COLUMN `col1` DATE NULL,
    DROP COLUMN `col2`,
    ADD COLUMN `col2` DATETIME(0) NULL,
    DROP COLUMN `col3`,
    ADD COLUMN `col3` JSON NULL,
    DROP COLUMN `col4`,
    ADD COLUMN `col4` TIMESTAMP(0) NULL;
```